News-Android-App/src/main/java/de/luhmer/owncloudnewsreader/NewsDetailFragment.java

```diff
@@ -299,8 +319,119 @@ public boolean onTouch(View v, MotionEvent event) {
 	}
 
 
+    private URL downloadUrl;
+    private long downloadID;
+    private DownloadManager dlManager;
+    private BroadcastReceiver downloadCompleteReceiver;
+
+    public void onCreateContextMenu(ContextMenu menu, View v, ContextMenu.ContextMenuInfo menuInfo) {
+        if (v instanceof WebView) {
+            WebView.HitTestResult result = ((WebView) v).getHitTestResult();
+            if (result != null) {
+                int type = result.getType();
+                if (type == WebView.HitTestResult.IMAGE_TYPE || type == WebView.HitTestResult.SRC_IMAGE_ANCHOR_TYPE) {
+                    String imageUrl = result.getExtra();
+                    if (imageUrl.startsWith("http")) {
```

Comment:
should work with https too, because it says: `startsWith("http")`, hence also includes https?!